### DISCUSSION TITLE
Documentation bug in error handling tutorial?

### DISCUSSION THREAD

**Discussion by VorpalBlade** [winnow-rs/winnow#544] [Q&A] [answered]
https://docs.rs/winnow/0.6.13/winnow/_tutorial/chapter_7/index.html uses `PResult<T>` (which is `Result<T, ErrMode<ContextError>>` by default), then in the section "Error Adaptation and Rendering" the manual suddenly switches to using `ParseError<&str, ContextError>`.

Nowhere is there an explaination of how to get from `ErrMode<ContextError>` to  `ParseError<&str, ContextError>` from what I can see.

**Comment by epage** [comment, marked as answer]
[Chapter 6](https://docs.rs/winnow/0.6.13/winnow/_tutorial/chapter_6/index.html) states

> Wraps the error in [ParseError](https://docs.rs/winnow/0.6.13/winnow/error/struct.ParseError.html)

It is a bit buried in the details.  if there are thoughts on how to improve it, I'm open.

**Reply by VorpalBlade** [reply]
Thanks. I think adding a cross reference or two would help. I'll make a PR after the holidays (midsummer in Sweden is on par with Christmas for importance and being busy with other things).
